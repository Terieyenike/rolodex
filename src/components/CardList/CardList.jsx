import React from 'react';
import { Card } from '../../components/Card/Card';
import style from '../../styles.module.css';

export const CardList = ({ monsters }) => (
  <section className={style.card__section}>
    <div className={style.container}>
      <div className={style.card}>
        {monsters.map((monster) => (
          <Card
            key={monster.id}
            name={monster.name}
            email={monster.email}
            company={monster.company.catchPhrase}
            id={monster.id}
          />
        ))}
      </div>
    </div>
  </section>
);
