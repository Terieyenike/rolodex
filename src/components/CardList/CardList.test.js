import React from 'react';
import { shallow } from 'enzyme';
import { CardList } from './CardList';

describe('CardList', () => {
  it('expect to render the CardList component', () => {
    const mockRobots = [
      {
        id: 1,
        name: 'Teri',
        email: 'teri@gmail.com',
        company: 'Transcorp Hilton',
      },
    ];
    expect.assertions(1);
    expect(shallow(<CardList monsters={mockRobots} />)).toMatchSnapshot();
  });
});
