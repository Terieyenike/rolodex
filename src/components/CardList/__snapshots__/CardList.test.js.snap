// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList expect to render the CardList component 1`] = `ShallowWrapper {}`;
